code/__HELPERS/_lists.dm

```diff
@@ -328,6 +328,11 @@
 		//Results will have a length of quality
 	return results
 
+//Pick a random element from the list.
+/proc/pick_random(list/L)
+	if(L.len)
+		var/picked = rand(1,L.len)
+		. = L[picked]		//Cut is far more efficient that Remove()
```

Comment:
You can just use pick(L) , it will automatically pick a random item from the list , no need to make a custom function.